site/student-lectures/cultural-algorithms/ca.py

```diff
@@ -0,0 +1,141 @@
+import math
```

Comment:
1. If you want to add code to the repo, put it under `src`. 
2. Change the name to "cultural_algorithm" or something like that 
3. You need to have unit tests for any code you add to the repo; we can't add untested code to the repo.

Reply:
Already made changes and the details are listed below